pkg/urbit/noun/imprison.c

```diff
@@ -478,7 +478,7 @@ u3i_list(u3_weak one, ...);
     return cut_t ? cut_w : i_w;
   }
 
-  __attribute__((no_sanitize("address")))
+NO_SANITIZE_ADDRESS
```

Comment:
GCC style `__attribute__((no_sanitize_address))` can be used directly here and below.

Reply:
done